client/connector.go

```diff
@@ -198,6 +199,56 @@ func (c *connector) Insert(tableName string, columnNames []string, rows []Row, u
 	return numRows, nil
 }
 
+// hllHash computes hash value from original value
+func hllHash(dataType memCom.DataType, value interface{}) (uint64, error) {
+	var out [2]uint64
+	var ok bool
+	numBytes := 4
+	switch dataType {
+	case memCom.UUID:
+		out, ok = memCom.ConvertToUUID(value)
+		if ok {
+			return out[0] ^ out[1], nil
+		}
+	case memCom.Uint32:
```

Comment:
better have a common convert function instead of switch here

Reply:
Done